comp/banner.yaml

```
@@ -0,0 +1,25 @@
+summary: Design and order a banner for the $SRYYYY competition
+
+priority: should
```

As this is a sponsor perk, it should probably be a must

```suggestion
priority: must
```